routes/web.php

```diff
@@ -37,8 +37,8 @@
     event(new PresenceMessageNotification($request->channel, $request->message));
 });
 
-Route::get('/listen', function () {
-    return view('listen');
-})->name('listen');
+Route::get('/ChatRoom', function () {
+    return view('ChatRoom');
+})->name('ChatRoom');
```

Comment:
This route is now unused, can we remove it?

Reply:
Yes, makes sense. We can do it, I will make a commit.

Reply:
Thanks